receiver/googlecloudpubsubreceiver/README.md

```diff
@@ -20,6 +20,10 @@ The following configuration options are supported:
   a fallback, when no `content-type` attribute is present.
 * `compression` (Optional): The compression that will be used on received data from the subscription. When set it can 
   only be `gzip`. This will only be used as a fallback, when no `content-encoding` attribute is present.
+* `endpoint` (Optional): Override the default Pubsub Endpoint, useful when connecting to the PubSub emulator instance
+  or switching between [global and regional service endpoints](https://cloud.google.com/pubsub/docs/reference/service_apis_overview#service_endpoints).
```

Comment:
never used the regional endpoints, so I tried the out of curiosity. Verified the branch locally and worked (as well with a regional endpoint)